arroyo/processing/processor.py

```diff
@@ -218,24 +261,36 @@ def _run_once(self) -> None:
                             logger.info(
                                 "Paused for longer than %d seconds", LOG_THRESHOLD_TIME
                             )
```

Comment:
Your metrics will be much more effective than this. What about just removing this. We added it to troubleshoot a very specific issue and we never used this log since.